cmd/container-disk-v2alpha/main.c

```diff
@@ -104,6 +104,7 @@ int main(int argc, char **argv) {
             /* These options don’t set a flag.
                 We distinguish them by their indices. */
             {"copy-path",    required_argument, 0, 'c'},
+            {"no-op", 0, 0, 'n'},
```

Comment:
why do we need this? can't we just pass `exit 0` to the command of the container?

Reply:
ContainerDisks are based on scratch. We explicitly copy this binary to an emptyDir with an init container, so that we have something to execute.